### ISSUE TITLE
error when compile warp_by_flow

### ISSUE THREAD

**Issue by wxz1996** [sylqiu/tof_rgbd_processing#6]
when i compile warp_by_flow int tf1.15 with cuda10.0,  it remind me the error 
#include "tensorflow/core/util/cuda_kernel_helper.h" not exist 
it seems that tf1.15 had remove it, did you test it in any other tf version

**Comment by sylqiu** [comment]
Unfortunately I haven’t tried it with other versions of the TF.

**Comment by wxz1996** [comment]
@sylqiu thanks, i have another question, i download the dataset ToFFlyingThings3D.zip and unzip it, but there has no folder "nogt" and "real" ...  should i need to use  the script in "Tof_pbrt_generation.zip" to  generate "nogt" and "real" ?

**Comment by sylqiu** [comment]
They are loaders for loading data without ground truth for testing for instance on your own data. You can check the implementation in the data loader file.

**Comment by wxz1996** [comment]
@sylqiu got it, but i can't find the real data, 'RealData/vivo_data/test_vivo5/' and 'RealData/vivo_data/test_vivo5/' int your rep or google driver

**Comment by sylqiu** [comment]
Unfortunately, the real dataset cannot be released, however we do have a sample included in the drive.

**Comment by wxz1996** [comment]
@sylqiu but when i run " model.single_test()" and "dataset_name = nogt", "self.path = '/data/wenxuanzheng/ToFFlyingThings3D/' set in TestingConfig, it remind me loss real data, and i didn't find any sample end with  '_reg_ir.png' and "_reg_depth.raw",  is "nogt" real data sample?

**Comment by wxz1996** [comment]
···parser.add_argument('--dataset_name', type=str, default='simtof', help='dataset')
    parser.add_argument('--dataset2_name', type=str, default='none', help='dataset')
    parser.add_argument('--is_training', type=int, default=0, help='True:1 or False:0')
    parser.add_argument('--is_from_scratch', type=int, default=0, help='Pretraining ?')
    parser.add_argument('--pretrain_ckpt_path', type=str, default='none', help='path to weight file')
    parser.add_argument('--save', type=str, default='./Experiment/', help='where to save stuff')
    parser.add_argument('--sigma', type=str, default=0, help='perturbation strength with mvg_aug in testing mode, set 0 to use uniform perturbation, see loaders.py for definition')
    parser.add_argument('--mvg_aug', type=int, default=1, help='set 1 to use multiview geometry augmentation')···
and
···
class TestingConfig(object):
    def __init__(self):
        self.wlast = '/home/wenxuanzheng/pc_compression/tof_rgbd_processing/Experiment/checkpoint/ckpt'
        self.output_save_dir ='./results'
        self.bs = 1

#dataset path
        self.path = '/data/wenxuanzheng/ToFFlyingThings3D/'
        self.use_fifo = False
···
this is my default set, but it report cuda error"Check failed: cusolverDnCreate(&cusolver_dn_handle) == CUSOLVER_STATUS_SUCCESS Failed to create cuSolverDN instance.", and i don't know it is the compile error in custom tf_op or the  parameter set error

**Comment by sylqiu** [comment]
Hmm, this is strange and I never seen it before. You can try replacing the cuda op by implementing python code counterparts.

**Comment by wxz1996** [comment]
nToF is the folder of real data sample? but it is not end with _reg_depth.raw ...
